src/sentry/toolbar/views/login_success_view.py

```diff
@@ -7,15 +7,20 @@
 
 session_cookie_name = settings.SESSION_COOKIE_NAME
 
+# touch 123
+
 
 @region_silo_view
 class LoginSuccessView(OrganizationView):
     def get(self, request: HttpRequest, organization, project_id_or_slug):
+        delay_ms = int(request.GET.get("delay") or 3000)
         return self.respond(
             TEMPLATE,
             status=200,
             context={
-                "delay": int(request.GET.get("delay", 3000)),
+                "delay_sec": int(delay_ms / 1000),
+                "delay_ms": delay_ms,
                 "cookie": f"{session_cookie_name}={request.COOKIES.get(session_cookie_name)}",
+                "token": "",
```

Comment:
This is intentionally left blank. If we were passing some other kind of token around we could do it here, and later there's code to save it into local storage. But this isn't implemented right now as long as cookies are serving the same purpose.